sdk/python/tests/expediagroup/test_milvus_online_store.py

```diff
@@ -431,3 +455,81 @@ def test_milvus_online_write_batch(self, repo_config, caplog):
             collection = Collection(name=self.collection_to_write)
             progress = utility.index_building_progress(collection_name=collection.name)
             assert progress["total_rows"] == total_rows_to_write
+
+    def test_milvus_online_read(self, repo_config, caplog):
+
+        # Generating data to directly load into Milvus
+        data_to_load = [
+            [i for i in range(10)],
+            [i + 2000 for i in range(10)],
+            [[random.random() for _ in range(2)] for _ in range(10)],
+        ]
+
+        # Creating a common schema for collection to directly add to Milvus
+        schema = CollectionSchema(
+            [
+                FieldSchema("film_id", DataType.INT64, is_primary=True),
+                FieldSchema("film_date", DataType.INT64),
+                FieldSchema("films", dtype=DataType.FLOAT_VECTOR, dim=2),
+            ]
+        )
+
+        # Creating an end-to-end example with just using pymilvus to understand how it works
```

Comment:
comment should be removed or re-phrased to remove the part "understand how it works"